src/main/java/com/osiris/autoplug/client/tasks/updater/plugins/TaskPluginsUpdater.java

```diff
@@ -459,27 +449,6 @@ else if (code == SearchResult.Type.RESOURCE_NOT_FOUND)
 
     }
 
-    /**
-     * @return null if all keys of this map have usages below the minimum and
-     * the keys are "null" or "0".
-     */
-    private Entry getValidEntryWithMostUsages(HashSet<Entry> _set, int minUsages) {
-        Entry[] entries = new QuickSort().sort(_set.toArray(new Entry[0]), (thisEl, otherEl) -> {
-            int thisUsage = ((Entry) thisEl.obj).usage;
-            int otherUsage = ((Entry) otherEl.obj).usage;
-            return Integer.compare(thisUsage, otherUsage);
-        });
-        for (int i = entries.length - 1; i >= 0; i--) {
-            int usages = entries[i].usage;
-            // If the entry with most usages doesn't exceed the minimum we can directly return null
-            if (usages < minUsages) return null;
-            String key = entries[i].key;
-            if (!Objects.equals(key, "null") && !Objects.equals(key, "0"))
-                return entries[i];
-        }
-        return null;
-    }
-
```

Comment:
you can also remove the entry class at the bottom of this file, since you moved it to your new class